api/external/cfgmgmt/cfgmgmt.proto

```diff
@@ -17,7 +17,9 @@ import "api/external/cfgmgmt/response/errors.proto";
 
 import "api/external/common/query/parameters.proto";
 import "api/external/common/version/version.proto";
-import "api/external/common/response.proto";
+import "api/external/common/response.proto";	
```

Comment:
idk why I kept having a merge conflict with these lines 20 - 22 when trying to push this. Will have to come back and fix.